windows/src/engine/inst/components.wxs

```diff
@@ -137,10 +141,20 @@
         <File Id="keyman64.dll" Name="keyman64.dll" KeyPath="yes" Source="..\..\..\bin\engine\keyman64.dll" />
       </Component>
 
+      <Component Guid="*">
+        <File Id="keyman64_ver.dll" Name="keyman64-ver$(var.VERSION_WITH_TAG).dll" KeyPath="yes" Source="..\..\..\bin\engine\keyman64-ver$(var.VERSION_WITH_TAG).dll" />
+      </Component>
+
       <Component Guid="{1D136A8F-BC8C-432F-B82F-822EBBE986F2}">
         <File Id="keymanx64.exe" Name='keymanx64.exe' KeyPath="yes" Source="..\..\..\bin\engine\keymanx64.exe" />
       </Component>
 
+      <Component Id="Reg_KeymanVersion" Guid="*">
+                <RegistryValue KeyPath="yes" Root="HKLM" Key="Software\Keyman\Keyman Engine" Name="keyman32 name" Type="string" Value="keyman32-ver$(var.VERSION_WITH_TAG).dll" />
+                <RegistryValue Root="HKLM" Key="Software\Keyman\Keyman Engine" Name="keyman64 name" Type="string" Value="keyman64-ver$(var.VERSION_WITH_TAG).dll" />
+                <RegistryKey ForceCreateOnInstall="yes" ForceDeleteOnUninstall="yes" Root="HKLM" Key="Software\Keyman\Keyman Engine" />
```

Comment:
I'm not sure this `<RegistryKey>` entry makes sense here? We don't want to be forcing deletion of this key?

Reply:
Having a read of the definitions it seems even ForceCrateOnInstall Yes is not needed either as it is only if you are setting up an empty key I will remove it. Then check it all still works